src/ducks/loan/LoanListPage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { Section } from 'components/Section'
+import LoanProgress from 'ducks/loan/LoanProgress'
+import CompositeRow from 'cozy-ui/transpiled/react/CompositeRow'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import { Bold } from 'cozy-ui/transpiled/react/Text'
+import AccountIcon from 'components/AccountIcon'
+import withFilters from 'components/withFilters'
+import AccountDetailsHeader from 'ducks/account/AccountDetailsHeader'
+
+const DumbLoanListPage = props => {
+  const { filteringDoc, filterByDoc } = props
+  const accounts = filteringDoc.accounts.data
```

Comment:
At this point we need to be sure that we have a filteringDoc, do you think defensive code against the case that filteringDoc is null would be necessary ?

Reply:
Yes that's a good idea. But what do you think we should do if filteringDoc is null ?

Reply:
Show something like "No accounts" ?